src/styles/blocks/card.scss

```diff
@@ -0,0 +1,134 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  min-width: 1024px;
+  margin: auto;
+  padding: 50px;
+
+  width: getCardContainerWidth(
+    $product-card-width,
+    $number-of-columns,
+    $column-space
+  );
+}
+
+.card {
+  margin-right: $column-space;
+  margin-bottom: $rows-space;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  background-color: #fff;
+  width: $product-card-width;
+  border-radius: 5px;
+  border: 1px #f3f3f3 solid;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    display: flex;
+    align-self: center;
+    height: 134px;
+    padding: 32px 19px 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: var(--font-darkblue);
+    margin: 0 16px 4px;
```

Comment:
You don't really need to add a horizontal margin to each block. Just add a correct padding to the card container